ibm_catalog.json

```
@@ -19,35 +19,35 @@
         "terraform",
         "solution"
       ],
-      "short_description": "Creates and configures a Secrets Manager instance.",
-      "long_description": "This deployable architecture is used to provision and configure an [IBM Cloud Secrets Manager](https://www.ibm.com/products/secrets-manager) instance. Centrally manage your secrets in a single-tenant, dedicated instance. This Terraform-based automation is part of a broader suite of IBM-maintained Infrastructure as Code (IaC) asset collection, each following the naming pattern \"Cloud automation for *servicename*\" and focusing on single IBM Cloud service. These single-service deployable architectures can be used on their own to streamline and automate service deployments through an [IaC approach](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-understanding-projects), or assembled together into a broader [automated IaC stack](https://cloud.ibm.com/docs/secure-enterprise?topic=secure-enterprise-config-stack) to automate the deployment of an end-to-end solution architecture.",
+      "short_description": "Creates and configures an IBM Cloud Secrets Manager instance.",
```

```suggestion
      "short_description": "Creates and configures a Secrets Manager instance.",
```

We're already in IBM Cloud when people see this, I would hesitate to mention it as often as we do to make it easier to read.